scene/gui/item_list.cpp

```diff
@@ -1541,6 +1547,10 @@ void ItemList::_get_property_list(List<PropertyInfo> *p_list) const {
 		pi.usage &= ~(get_item_icon(i).is_null() ? PROPERTY_USAGE_STORAGE : 0);
 		p_list->push_back(pi);
 
+		pi = PropertyInfo(Variant::BOOL, vformat("item_%d/selectable", i));
+		pi.usage &= ~(is_item_selectable(i) ? PROPERTY_USAGE_STORAGE : 0);
```

Comment:
Shouldn't this be 
```suggestion
		pi.usage &= ~(!is_item_selectable(i) ? PROPERTY_USAGE_STORAGE : 0);
```
Like for `disabled`?

IIUC this will store the value when `selectable` is enabled (default) and not when disabled. It should be the other way around (store non-default).

Reply:
Nevermind I think I misread, this is probably good as is.

Reply:
In my first try I've done the same as disabled. But it was not working : By default items were selectable but the switch checkbox didn't have any effects at first uncheck. (need to uncheck, check and uncheck).

I though it was because selectable is true by default whereas disabled is false by default so this line need to be inverted.

After changing this line, all was working as expected.

So, are you sure it need to be changed ?